agri-store-react/src/Components/Context/CartContext.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import { ProductListingPagecontext } from "./ContextProductListingPage";
+import axios from "axios";
+function CartContext() {
+  const { productdata, setProductData } = useContext(ProductListingPagecontext);
+
+  const addToCart = async (productdata, setCart) => {
+    const response = await axios({
+      method: "POST",
+      url: `/api/user/cart`,
+      headers: { authorization: localStorage.getItem("token") },
+      data: { product: productdata },
+    });
+    console.log(response.data.cart);
+    setCart(response.data.cart);
+  };
+
+  const addToWishlist = async (productdata, setWish) => {
+    const response = await axios({
+      method: "POST",
+      url: `/api/user/wishlist`,
+      headers: { authorization: localStorage.getItem("token") },
+      data: { product: productdata },
+    });
+    console.log(response.data.wishlist);
```

Comment:
here also

Reply:
removed all the possible console.logs , only few are that which supports error handling when the error handling states will be defined then these logs will too removed